amd_openvx_extensions/amd_media/decoder.cpp

```diff
@@ -723,10 +733,16 @@ void CLoomIoMediaDecoder::DecodeLoop(int mediaIndex)
                     eof[mediaIndex] = true;
                     PushAck(mediaIndex, -1);
                     av_packet_unref(&avpkt);
+#if ENABLE_PERF_MEASURE
+                    std::cout << "AMD_MEDIA_DECODER Time (ms): " << totalDecodeTime.count() * 1000 / frameno << std::endl;
```

Comment:
add the below line after line 737 to report the total transfer time
std::cout << "average transfer time per frame (ms): " << totalTransferTime.count() * 1000 / frameno << std::endl;